processors.go

```diff
@@ -198,6 +199,15 @@ func ProcessorCompanyName(cmap *ColumnMapper, input string) (string, error) {
 	return fake.Company(), nil
 }
 
+// ProcessorRandomBoolean will return a random boolean value.
+func ProcessorRandomBoolean(cmap *ColumnMapper, input string) (string, error) {
```

Comment:
What are your thoughts about renaming this function to ProcessorRandomBooleanString? My concern here is that function is returning a string representation of a boolean and not an actual Go bool type.  If we end up adding a ProcessorRandomBoolean in the future that returns a bool things may get a little confusing.

Reply:
Bah I see what you are doing now. Ignore my above comments.